### DISCUSSION TITLE
Discord picture wont work

### DISCUSSION THREAD

**Discussion by Hauntinq** [jojo2357/Music-Discord-Rich-Presence#18] [Q&A] [answered]
I've followed all the steps, yet it wont work.
![Opera Snapshot_2021-03-13_171755_discord com](https://user-images.githubusercontent.com/80594616/111038372-71eb5480-8420-11eb-8dfb-5398d326a959.png)
![Opera Snapshot_2021-03-13_171818_discord com](https://user-images.githubusercontent.com/80594616/111038373-7283eb00-8420-11eb-82f5-0e586b3f1b8d.png)
![Screenshot (75)](https://user-images.githubusercontent.com/80594616/111038374-731c8180-8420-11eb-8837-41be1ab56b6d.png)
![Screenshot (76)](https://user-images.githubusercontent.com/80594616/111038375-731c8180-8420-11eb-9caa-25274a34ecfa.png)

**Comment by jojo2357** [comment, marked as answer]
The key (the part after the =) has to match what the asset is named in discord (in this case yandhi)

**Reply by Hauntinq** [reply]
I've changed it and it still doesn't work
![Screenshot (78)](https://user-images.githubusercontent.com/80594616/111039199-b7118580-8424-11eb-9647-b52016f974a1.png)

**Reply by jojo2357** [reply]
Not there! After the album on line 3.

**Reply by Hauntinq** [reply]
Thank you so much!!
